extensions/src/main/java/org/pentaho/platform/plugin/services/metadata/PentahoMetadataDomainRepository.java

```diff
@@ -14,15 +14,17 @@
  * See the GNU Lesser General Public License for more details.
  *
  *
- * Copyright (c) 2002-2019 Hitachi Vantara. All rights reserved.
+ * Copyright (c) 2002-2022 Hitachi Vantara. All rights reserved.
  *
  */
 
 package org.pentaho.platform.plugin.services.metadata;
 
 import org.apache.commons.io.IOUtils;
-import org.apache.commons.lang.StringEscapeUtils;
-import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang3.BooleanUtils;
```

Comment:
FWIW, I noticed that a `PentahoMetadataDomainRepositoryIT` class exists.

Reply:
I ran them locally, save results on master and this PR branch

_[ERROR] Tests run: 24, Failures: 3, Errors: 2, Skipped: 0_

There isn't a jenkins job for than this project, but there is one for pentaho/pentaho-platform on the list
https://thanos.orl.eng.hitachivantara.com/job/9.3/job/it-snapshot-jobs/

however this one for pentaho/pentaho-platform-ee and it currently passing for 9.3.

I don't think 9.4 it builds are up https://thanos.orl.eng.hitachivantara.com/job/9.4/